packages/frint-react/src/components/Region.js

```diff
@@ -56,13 +57,15 @@ export default class Region extends React.Component {
       return null;
     }
 
+    const { className } = this.props;
+
     return (
-      <div>
+      <div className={className}>
         {listForRendering.map((item) => {
           const { Component, name } = item;
 
           return (
-            <Component key={`app-${name}`} />
+            <Component className={className} key={`app-${name}`} />
```

Comment:
we can remove the `className` prop from here, and keep it only in the parent element.

Reply:
Done!